pkg/service/v0/service.go

```diff
@@ -115,14 +115,15 @@ func getFailsafeIdentifier(c context.Context, identifier *proto.Identifier) *pro
 		identifier = &proto.Identifier{}
 	}
 	if identifier.AccountUuid == "me" {
-		ownAccountUUID := c.Value(middleware.UUIDKey).(string)
-		if len(ownAccountUUID) > 0 {
-			identifier.AccountUuid = ownAccountUUID
-		} else {
-			// might be valid for the request not having an AccountUuid in the identifier.
-			// but clear it, instead of passing on `me`.
-			identifier.AccountUuid = ""
+		ownAccountUUID, ok := c.Value(middleware.UUIDKey).(string)
+		if ok {
```

Comment:
Can we have this as `if ok && len(ownAccountUUID) > 0 {` instead of two separate `if`s? or is there a golang-reason against it?

Reply:
I was thinking if we have to check that the string has a value at all. Can it be set to `""`??